pkg/volume/util/atomic_writer.go

```diff
@@ -63,8 +63,9 @@ type AtomicWriter struct {
 
 // FileProjection contains file Data and access Mode
 type FileProjection struct {
-	Data []byte
-	Mode int32
+	Data   []byte
+	Mode   int32
+	FsUser *int64
```

Comment:
Does kubelet ever run as non-root?

Reply:
I'm going with probably somewhere, someone is doing this. Nevermind.

Reply:
atomic writer should have the assumption that kubelet runs as root to chmod, chmod etc at least for some period of time.

Reply:
Actually, right now if kubelet runs as non-root, files written by atomic writer will be the some random user. We should probably always Chown() files. I suggest changing FsUser to an int64 (defaults to 0 which is root) and always chowning. Might want to do the same for group as well.

Reply:
Kubelet could possible run as non-root with CAP_FOWNER|CAP_CHOWN.

Reply:
we might consider doing this in another PR on all volume types, because the change to atomic writer only applies to ephemeral volumes and will cause inconsistency if a container mounts other volume types together with projected token volume.

Reply:
Cleanup for a later day. I'm going to resolve.